rtmbot.py

```diff
@@ -55,12 +55,19 @@ def output(self):
         for plugin in self.bot_plugins:
             limiter = False
             for output in plugin.do_output():
-                channel = self.slack_client.server.channels.find(output[0])
-                if channel != None and output[1] != None:
+                destination = output[0]
+                message = output[1]
+                # things that start with U are users. convert to an IM channel.
+                if destination.startswith('U'):
+                    result = json.loads(self.slack_client.api_call('im.open', user=destination))
+                    channel = self.slack_client.server.channels.find(result[u'channel'][u'id'])
```

Comment:
These would benefit from  either a `result.get(u'channel', {}).get(u'id', None)` or a `try/except` for `KeyError`